internal/auth/authorizations.go

```diff
@@ -108,6 +117,7 @@ func QueryAuthd(authd, remoteIP, tlsEnabled, authSecret string) (*State, error)
 			switch p {
 			case "subscribe", "publish":
 			default:
+				// TODO: Can we add more granular permissions, for example delete, pause, etc?
 				return nil, fmt.Errorf("unknown permission %s", p)
 			}
```

Comment:
I think logging is sufficient. I think the current implementation of erroring here is unnecessary, and the logging might help debug situations where you're not getting the permissions you expect (typos, etc.).